dev/bots/README.md

```diff
@@ -119,6 +119,11 @@ Instructions to update the Android Tools version that the bots download by execu
 3. Run upload_android_tools.py -t sdk
    `$ dev/bots/upload_android_tools.py -t sdk`
 
+4. Run `dev/bots/android_tools/sdk/tools/bin/sdkmanager --update`. On Windows, run `sdkmanager.bat` instead. If the
```

Comment:
Can you also update the versions mentioned a few lines above?

Reply:
Can I delete them instead? They seem redundant as it's easy to get the current versions by running `sdkmanager --list` and it's (apparently) easy to forget to update this list here.

Reply:
Yeah, that is good as well. Maybe you can document how to get the latest version from sdkmanager instead?

Reply:
Done.